lib/RelatedResourceProviders/TalkRelatedResourceProvider.php

```diff
@@ -143,9 +150,16 @@ private function convertToRelatedResource(TalkRoom $share): IRelatedResource {
 
 		$related = new RelatedResource(self::PROVIDER_ID, (string)$share->getRoomId());
 		$related->setTitle($share->getRoomName())
-				->setSubtitle('Talk Room')
-				->setTooltip('Talk Room \'' . $share->getRoomName() . '\'')
-				->setUrl($url)
+				->setSubtitle($this->l10n->t('Talk'))
+				->setTooltip($this->l10n->t('Talk Conversation \'%s\'', $share->getRoomName()))
```

Comment:
```suggestion
				->setTooltip($this->l10n->t('Talk conversation \'%s\'', $share->getRoomName()))
```
From @nickvergessen in https://github.com/nextcloud/nextcloud-vue/pull/3236#discussion_r969291882